.eslintrc.json

```diff
@@ -82,6 +81,7 @@
         "mocha": true
       },
       "rules": {
+        "@typescript-eslint/no-invalid-void-type": "off",
```

Comment:
The only occurrence in the tests is here:
https://github.com/vaadin/web-components/blob/725926c58c4f07246a6c05160e2472ce1700fe23/packages/virtual-list/test/typings/virtual-list.types.ts#L34

Not sure what the purpose of that line is, but maybe we can change that to just:
```
virtualList.scrollToIndex(3)
```
which would allow us to remove the ESlint override.

Reply:
Good catch, but instead we probably should use something like this:

```ts
assertType<(number) => void>(virtualList. scrollToIndex);
```

Reply:
Makes sense 👍

Reply:
Good catch, thanks! Let me update the PR.